Documentation/network/pod-mac-address.rst

```diff
@@ -0,0 +1,69 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _pod_mac_address:
+
+************************************
+Use a Specific MAC Address for a Pod
+************************************
+
+
+Some applications bind software licenses to network interface MAC addresses.
+Typically, the solution is very complex and requires things like Docker within
+the cluster.
+
+Therefore, cilium provides the ability to specific MAC addresses
+for pods instead of letting the operating system allocate them.
```

Comment:
```suggestion
Cilium provides the ability to specific MAC addresses for pods
at deploy time instead of letting the operating system allocate them.
```